vmb/matrimony/models/profiles.py

```diff
@@ -1698,3 +1698,10 @@ class MatrimonyProfileStats(BaseModel):
 
     class Meta:
         db_table = "matrimony_profile_stats"
+
+from tos.models import UserAgreement
```

Comment:
A better way will be to create a new app called `tos_custom` to do this monkey patching

Reply:
Addressed and Resolved.